hooks/pre-command

```diff
@@ -29,15 +23,17 @@ image_tag="$(compute_tag ${docker_file})"
 
 if ! docker pull "${image_name}:${image_tag}"; then
   echo "Image not cached, building"
+  docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" || exit 1
   if [ "$BUILDKITE_BRANCH" == "$BUILDKITE_PIPELINE_DEFAULT_BRANCH" ]; then
-    docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" -t "${image_name}:latest" || exit 1
-  else
-    docker build . --file "${docker_file}" -t "${image_name}:${image_tag}" || exit 1
+    docker tag "${image_name}" "latest"
+    docker push "${image_name}" "latest"
```

Comment:
Should this be

```shell
docker tag "${image_name}:${image_tag}" "${image_name}:latest"
docker push "${image_name}:latest"
```

- https://docs.docker.com/engine/reference/commandline/tag/
- https://docs.docker.com/engine/reference/commandline/push/